embuilder.py

```diff
@@ -56,6 +56,8 @@
     'libemmalloc-memvalidate',
     'libemmalloc-verbose',
     'libemmalloc-memvalidate-verbose',
+    'libmimalloc',
```

Comment:
Is there any reason someone would want to use mimalloc instead of dlmalloc when not building with threads?

Reply:
It is actually faster even with a single core, see the first chart. (But it is substantially larger so I doubt it would be a common thing.)